.github/workflows/test.yml

```diff
@@ -19,6 +19,7 @@ jobs:
         with:
           otp-version: 22.2
           elixir-version: 1.10.4
+          experimental-otp: true
```

Comment:
Could you share with us What does this one do @hanam1ni ?

Reply:
It's a temporary solution, there is an issue on Github action when installing the Elixir the error about `SSL` is raised because Ubuntu version of CI machine. So this one will be removed in the future. You can check here for more information https://github.com/actions/setup-elixir/issues/31 🙌

Reply:
Thanks for sharing that @hanam1ni I thought that is something nice so we could consider to add it into the Elixir template :D